app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Ingredients</h1>
+
+<ul>
+    <% @ingredients.each do |ingredient| %>
+        <li><%= "#{ingredient.name}: #{ingredient.cost}" %></li>
+    <% end %>
+</ul>
```

Comment:
Another way to do this is: `<p><%= ingredient.name %>: <%= ingredient.cost %></p>`.

What you have also works, but just want to show you a different approach.